src/test/java/com/glovoapp/planningpoker/e2e/voting/Median2UsersTest.java

```diff
@@ -0,0 +1,60 @@
+package com.glovoapp.planningpoker.e2e.voting;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Optional;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+public class Median2UsersTest {
+    private final WebDriver driver = getDriver();
+    private final WebDriver driver2 = getDriver();
+
+    MainPage firstUserMainPage = new MainPage(driver);
+    SessionPage firstUserSessionPage = new SessionPage(driver);
+    MainPage secondUserMainPage = new MainPage(driver2);
+    SessionPage secondUserSessionPage = new SessionPage(driver2);
+
+    private final String sessionName = "testM";
+    private final String userName1 = "michal";
+    private final String userName2 = "pawel";
+    private final String expectedResult = "Median: 3.5";
+
+    @AfterEach
+    void releaseDriver() {
+        WebDriverFactory.releaseDriver(driver);
+        WebDriverFactory.releaseDriver(driver2);
+    }
+
+    @Test
+    void afterVotingFor2And5_TheMedianResultShouldBeAsExpected() {
+
+// ------------------------- FirstUser ----------------------
+        firstUserMainPage.openGlovoSite();
+        firstUserMainPage.joinSession(sessionName);
+        firstUserSessionPage.enterName(userName1);
+        //Refreshing site
+        firstUserSessionPage.specClick();
+        firstUserSessionPage.specClick();
+        Optional<WebElement> firstButton = firstUserSessionPage.getVotingButton("2");
+        firstButton.get().click();
```

Comment:
[IDE] Please pay more attention to [your IDE warnings](https://www.jetbrains.com/help/idea/code-inspection.html). This code shows:

> 'Optional.get()' without 'isPresent()' check 

ᴛʜɪs ᴀᴄᴛɪᴏɴ ᴡᴀs ᴘᴀʀᴛɪᴀʟʟʏ ᴀᴜᴛᴏᴍᴀᴛᴇᴅ ʙʏ [GɪᴛHᴜʙ's ʀᴇsᴘᴏɴsᴇ ᴛᴇᴍᴘʟᴀᴛᴇs](https://github.com/settings/replies)